pages/api_trading.md

```diff
@@ -32,8 +32,30 @@ These are orders which have a finite lifetime and are valid for a short period o
 
 All the below orders are transactions and are signed. The client[4](https://docs.dydx.exchange/developers/clients/validator_client) can be used or the transaction can be placed manually by following conforming to the formats specified here[5](https://github.com/dydxprotocol/v4-chain/blob/main/proto/dydxprotocol/clob/tx.proto). For manual transactions, the transaction can be signed using the following documentation[6](https://docs.cosmos.network/v0.50/user/run-node/txs).
 #### Place
+To place an order, refer to the proto here[7](https://github.com/dydxprotocol/v4-chain/blob/main/proto/dydxprotocol/clob/tx.proto#L27) to understand the parameters required.
```

Comment:
Think this could use more detail. Once you build the order proto, what do you do with it? Does it go inside another proto? How to sign it? Where to broadcast it? (I think there's a REST endpoint for this too, just like the other full node endpoints.) 

It's okay if we don't show them how step by step, but it would be great if a reader could read this section and follow links to everything they need to implement this in whatever language they're using.